iac-common/src/test/java/org/sonar/iac/common/extension/visitors/InputFileContextTest.java

```diff
@@ -208,4 +210,28 @@ void shouldFallbackToBeginOfTheFileWhenInvalidLocation() {
     assertThat(logTester.logs(Level.DEBUG))
       .contains("Error when creating valid line offset of pointer, fallback to beginning of the file.");
   }
+
+  @Test
+  void shouldReportTwoIssuesEvenIfThyHaveTheSameHash() {
+    var rule1 = RuleKey.of("kubernetes", "S6897");
+    var textRange1 = TextRanges.range(10, 6, 10, 10);
+    var rule2 = RuleKey.of("kubernetes", "S6864");
+    var textRange2 = range(13, 6, 13, 10);
+    var hash1 = Objects.hash(rule1, textRange1, List.of());
+    var hash2 = Objects.hash(rule2, textRange2, List.of());
+    // Those 2 Issue reports have the same hash
+    assertThat(hash1).isEqualTo(hash2);
```

Comment:
So it's "luck" that they have the same hash?

Reply:
Yes, it is "luck". And it was here for a long time ;-)